src/Headless/Avalonia.Headless/AvaloniaHeadlessPlatform.cs

```diff
@@ -67,9 +67,13 @@ public HeadlessWindowingPlatform(PixelFormat frameBufferFormat)
         
         internal static void Initialize(AvaloniaHeadlessPlatformOptions opts)
         {
+            var clipboardImpl = new HeadlessClipboardImplStub();
+            var clipboard = new Clipboard(clipboardImpl);
+
             AvaloniaLocator.CurrentMutable
                 .Bind<IDispatcherImpl>().ToConstant(new ManagedDispatcherImpl(null))
-                .Bind<IClipboard>().ToSingleton<HeadlessClipboardStub>()
+                .Bind<IClipboardImpl>().ToConstant(clipboardImpl)
+                .Bind<IClipboard>().ToConstant(clipboard)
```

Comment:
Since `Clipboard` type is the same on all platforms, do we need to register it via locator? As an alternative, it can be created in the application, when requested.